src/blockdev.rs

```diff
@@ -976,6 +976,28 @@ pub fn is_dasd(device: &str) -> Result<bool> {
     Ok(target.to_string_lossy().starts_with("/dev/dasd"))
 }
 
+/// Checks if virtio-blk device is backed by an IBM DASD disk
+#[cfg(target_arch = "s390x")]
+pub fn is_vda_dasd(device: &str) -> Result<bool> {
+    let target =
+        canonicalize(device).with_context(|| format!("getting absolute path to {}", device))?;
+    let valid;
+    let cmd = Command::new("fdasd").arg("-i").arg(target).output()?;
```

Comment:
It might be good to check whether fdasd is available.

Reply:
correct. it is actually not available and would need to be added here: https://github.com/coreos/fedora-coreos-config/blob/testing-devel/overlay.d/05core/usr/lib/dracut/modules.d/40ignition-ostree/module-setup.sh

Reply:
It is available as part of s390x-utils.

Reply:
ah..my bad..that was for the initramfs...yeah all good then